legacy/core/src/test/java/com/fsck/k9/preferences/SettingsImporterTest.kt

```diff
@@ -154,6 +161,101 @@ class SettingsImporterTest : K9RobolectricTest() {
         }
     }
 
+    @Test
+    fun `importSettings()  configures unifiedInbox when globalSettingsImported is false`() {
+        val unifiedInboxConfigurator = mock<UnifiedInboxConfigurator>()
+        val generalSettingsWriter = mock<GeneralSettingsWriter>()
+        val settingsFileParser = mock<SettingsFileParser>()
+
+        val settingsImporter = SettingsImporter(
+            settingsFileParser = settingsFileParser,
+            generalSettingsValidator = mock<GeneralSettingsValidator>(),
+            accountSettingsValidator = mock<AccountSettingsValidator>(),
+            generalSettingsUpgrader = mock<GeneralSettingsUpgrader>(),
+            accountSettingsWriter = mock<AccountSettingsWriter>(),
+            accountSettingsUpgrader = mock<AccountSettingsUpgrader>(),
+            generalSettingsWriter = generalSettingsWriter,
+            unifiedInboxConfigurator = unifiedInboxConfigurator,
+        )
```

Comment:
This is making the test much more complicated than it needs to be. The only dependency that needs to be replaced is `UnifiedInboxConfigurator`. The rest can be retrieved via Koin.

```kotlin
import org.koin.test.get

val unifiedInboxConfigurator = mock<UnifiedInboxConfigurator>()

val settingsImporter = SettingsImporter(
    settingsFileParser = get(),
    generalSettingsValidator = get(),
    accountSettingsValidator = get(),
    generalSettingsUpgrader = get(),
    accountSettingsWriter = get(),
    accountSettingsUpgrader = get(),
    generalSettingsWriter = get(),
    unifiedInboxConfigurator = unifiedInboxConfigurator,
)
```

Reply:
@cketti Done!